psse/psse-converter/src/test/java/com/powsybl/psse/converter/PsseImporterTest.java

```diff
@@ -0,0 +1,41 @@
+package com.powsybl.psse.converter;
+
+/**
+ * Copyright (c) 2019, RTE (http://www.rte-france.com)
```

Comment:
Licence header should be before the package declaration
year is wrong

Reply:
ok